### PR TITLE
Build the covid-19-app

### PR DESCRIPTION
**Pull request by Christabelle143**
# For this React capstone project I build a mobile web application to check a list of metrics (numeric values) that we create making use of React and Redux.
-Select an API that provides numberic data about a topic that I liked
- Build one page with a list of items that could be filtered by some parameters
- Build one page for the item details
Home page
  - When the page loads, the webapp shows the list of categories that could be filtered out by some parameter.
  - Along with the category name, you will display some numeric values per category that come from the API.
  - When the user clicks (or taps) on a category item, the application navigates to the details page.
Details page
  - In the details page, the webapp retrieves data from the API to show detailed data in the category.
  - When the user clicks on the "Back" button (<), the user navigates to the home page.
  - Create unit tests for pure functions in your Redux code.
  - Create integration tests for your application using the React Testing Library.
- Style my components to match the design provided.
- Add search filter bar
- Deploy the project and test for final details.
- Record a video for your project.
- Create a good README and PR description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,11 +38,11 @@ You will also see any lint errors in the console.
 
 ## Live Demo Link
 
-### [Live Demo]
+### [Live Demo](https://christabelle143-78cc21.netlify.app/)
 
 ## Loom Video Link
 
-### [Loom]
+### [Loom](https://www.loom.com/share/e826ba201e4146508c393a459515834c)
```
Comment (rloterh) [inline README.md]: Good job with project presentation, however, you missed some steps in your video presentation. Your video presentation only showed functional aspect of your app and failed to highlight any portion of your code. 
- [ ] Kindly ensure that the video contains a highlight about a piece of the code that you find interesting or that you were proud of making.

File: README.md [hunk 2]
```diff
@@ -38,11 +38,11 @@ You will also see any lint errors in the console.
...
```
Comment (rloterh) [inline README.md:47]: - [ ] It's a nice and smart thing to give appropriate credit to **Nelson Sakwa**, the author of [the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly ensure to add this under the **Acknowledgements** section of your ReadMe.

File: README.md [hunk 3]
```diff
@@ -38,11 +38,11 @@ You will also see any lint errors in the console.
...
```
Comment (rloterh) [inline README.md:48]: - [ ] OPTIONAL
MIT license link in your ReadMe on _line 140_ is broken, kindly endeavor to fix it.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Christabelle, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍
- State management with Redux properly implemented
- API fetch properly implemented
- Navigation working as intended
- Filter works great
- Good unit test coverage 
- Awesome UI design
- Good ReadMe

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: APPROVED 🟢🟢🟢 

Hi @Christabelle143,

Good job on making the changes asked by the previous reviewer 👍 
Your project is complete!  

## Highlights:
✅ You used Gitflow correctly
✅ Linter checks passed
✅ README is professional 👍 
✅ Good descriptive PR 👍 
✅ App works fine as expected 👍 

## Recommendation Change

- [ ] It would be nice to remove the left arrow on the navbar side of 2021 because when it's clicked, it takes you nowhere. You already have that arrow on the details page and exactly where it's meant to be. I would suggest you give a title of your app as the logo and replace 2021. You could use, AfricaCovidMetrics or something nice to serve as your logo 👍 

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**